tests/impl.cpp

```diff
@@ -2502,6 +2502,49 @@ result_t test_mm_xor_ps(const SSE2NEONTestImpl &impl, uint32_t i)
                          *((float *) &d2), *((float *) &d3));
 }
 
+result_t test_mm_set_rounding_mode(const SSE2NEONTestImpl &impl, uint32_t i)
+{
+    const float *_a = impl.mTestFloatPointer1;
+    result_t res_toward_zero, res_to_neg_inf, res_to_pos_inf, res_nearest;
+
+    __m128 a = do_mm_load_ps(_a);
+    __m128 b, c;
+    _MM_SET_ROUNDING_MODE(_MM_ROUND_TOWARD_ZERO);
+    b = _mm_round_ps(a, _MM_FROUND_CUR_DIRECTION);
+    c = _mm_round_ps(a, _MM_FROUND_TO_ZERO | _MM_FROUND_NO_EXC);
+    res_toward_zero =
+        validateFloatEpsilon(c, ((float *) &b)[0], ((float *) &b)[1],
+                             ((float *) &b)[2], ((float *) &b)[3], 5.0f);
+
+    _MM_SET_ROUNDING_MODE(_MM_ROUND_DOWN);
+    b = _mm_round_ps(a, _MM_FROUND_CUR_DIRECTION);
+    c = _mm_round_ps(a, _MM_FROUND_TO_NEG_INF | _MM_FROUND_NO_EXC);
+    res_to_neg_inf =
+        validateFloatEpsilon(c, ((float *) &b)[0], ((float *) &b)[1],
+                             ((float *) &b)[2], ((float *) &b)[3], 5.0f);
```

Comment:
What is the reason of setting the value to `5.0f`?

Reply:
Not in particular, I took the same value than the test_mm_round_ps